scss/override/_reboot.scss

```diff
@@ -0,0 +1,8 @@
+// Overrides to Reboot styles
+
+/*
+* > LINKS
+*/
+a {
+  font-weight: $font-weight-bold;
```

Comment:
This (I think) causes the sidebar menu items to be bold on the docs. Is there a way to leave the weight of the menu items in the docs the way they were?

Reply:
I have added a commit to restore the initial font-weight for sidebar links in our _custom-docs.scss file (where all customizations related to the docs should go).